frontend/src/App.js

```diff
@@ -6,19 +6,27 @@ import AdminTicketTypes from './components/TicketTypes/AdminTicketTypes';
 import Login from './components/Login/Login';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import PrivateRoute from './components/PrivateRoute/PrivateRoute';
+import { Provider } from 'react-redux';
+import store from './flux/store';
 
 function App() {
+  /* useEffect(() => {
+    store.dispatch(loadUser()); --- for when we have an actual endpoint for it...
+  }, []); */
```

Comment:
@marcianszlovak `useDispatch` is nicer if we want to do this here, but I'm not sure